Tophat/Utilities/ArtifactDownloader.swift

```diff
@@ -46,11 +46,18 @@ final class ArtifactDownloader: NSObject {
 		do {
 			defer { progressObservation = nil }
 
-			let (localURL, _) = try await URLSession.shared.download(from: url, delegate: self)
+			let (localURL, response) = try await URLSession.shared.download(from: url, delegate: self)
+			let statusCode = (response as? HTTPURLResponse)?.statusCode ?? 0
+			if statusCode != 200 {
+				let body = (try? String(contentsOf: localURL)) ?? "<no response>"
```

Comment:
How valuable is the actual response data?  It does seem to clutter the alert quite a bit—is the status code enough?  To get more granular I wonder if it could be enough to look at the logs _via_ Console.  But let me know what you think.